src/Chainweb/Version/Mainnet.hs

```diff
@@ -139,7 +139,9 @@ mainnet = ChainwebVersion
         Chainweb217Pact -> AllChains (ForkAtBlockHeight $ BlockHeight 3_250_348) -- 2022-12-02T00:00:00+00:00
         Chainweb218Pact -> AllChains (ForkAtBlockHeight $ BlockHeight 3_512_363) -- 2023-03-03 00:00:00+00:00
         Chainweb219Pact -> AllChains (ForkAtBlockHeight $ BlockHeight 3_774_423) -- 2023-06-02 00:00:00+00:00
-        Chainweb220Pact -> AllChains ForkNever
+        Chainweb220Pact -> AllChains (ForkAtBlockHeight $ BlockHeight 4_056_499) -- 2023-09-08 00:00:00+00:00
+        -- TODO: This is a placeholder!
+        Chainweb221Pact -> AllChains (ForkAtBlockHeight $ BlockHeight 4_156_499) -- 2023-10-14 00:00:00+00:00
```

Comment:
For this and testnet replace with `AllChains ForkNever` just to avoid accidentally leaving this in